virtool/blast/task.py

```diff
@@ -53,7 +53,7 @@ async def request(self):
 
         self.rid = blast.rid
 
-    async def wait(self):
+    async def wait(self, retries: int = 3):
```

Comment:
In normal usage this function will never be called with a value for `retries` will only ever by the default value. Take a look at `BaseTask` for a detailed understanding of why. I would personally hardcode it at 3.